core/rio/turtle/src/main/java/org/eclipse/rdf4j/rio/turtle/TurtleWriter.java

```diff
@@ -70,12 +85,6 @@
 	protected Resource lastWrittenSubject;
 	protected IRI lastWrittenPredicate;
 
-	/**
-	 * Always null
-	 */
-	@Deprecated
-	protected Model prettyPrintModel;
-
 	private final Deque<Resource> stack = new LinkedList<>();
 	private final Deque<IRI> path = new LinkedList<>();
```

Comment:
These should be ArrayDeque, unless we expect them to grow very very large (since expanding the ArrayDeque gets costlier the bigger it is, talking millions of entries).

Reply:
We've got a lot of pre-1.6 code in places :) Good point, will fix.